androidApp/src/test/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/splash/SplashScreenTest.kt

```diff
@@ -64,10 +73,41 @@ internal class SplashScreenTest {
         }
     }
 
+//    @Test
+//    fun `when logging in successfully, it navigates to the Home screen`() = initComposable {
+//        waitForAnimationEnd()
+//
+//        onNodeWithText(context.getString(R.string.login_email)).performTextInput("avishek@nimblehq.co")
+//        onNodeWithText(context.getString(R.string.login_password)).performTextInput("00000000")
+//        onNodeWithText(context.getString(R.string.login_button)).performClick()
+//
+//        verify { mockOnLoginSuccess() }
+//    }
```

Comment:
You need to wait for idle time since when you click on `performClick` it is still showing loading and hasn't return success yet. But Like I have suggested above ☝️ we need to reconsider about the navigation logic. Can you check with our android template for that? May be we can reference it.
```suggestion
@Test
    fun `when logging in successfully, it navigates to the Home screen`() = initComposable {
        waitForAnimationEnd()

        onNodeWithText(context.getString(R.string.login_email)).performTextInput("avishek@nimblehq.co")
        onNodeWithText(context.getString(R.string.login_password)).performTextInput("00000000")
        onNodeWithText(context.getString(R.string.login_button)).performClick()
        waitForIdle()

        verify { mockOnLoginSuccess() }
    }
```

Reply:
Fixed in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/95/commits/85ea0adaf6306ebbb911226778e242beaf6ea2fc